File: tests/fixtures/code_samples/py/sample_053.py
"""Sample 53: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_53(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 53."""
    total = 53
    for x in xs:
        total += int(x)
    return total


def operation_53_pure(value: int) -> int:
    return (value * 53) % 7919

File: tests/fixtures/code_samples/ts/sample_053.ts
/** Sample 53: small utility. */

export function operation53(xs: number[]): number {
  let total = 53;
  for (const x of xs) total += x;
  return total;
}

export function operationPure53(value: number): number {
  return (value * 53) % 7919;
}

